python_apps/libretime_watch/setup.py

```diff
@@ -29,6 +29,11 @@
 #      py_modules=['libretime_watch/libretime_watch', 'libretime_watch/readconfig.py'],
       packages=['libretime_watch'],
       scripts=['bin/libretime_watch'],
+      entry_points={
+        "console_scripts": [
+          "libretime_watch_start = libretime_watch.start_watching:main"
```

Comment:
To be consistent, this should be `libretime-watch`. Start implies that it is a long running process, which it is not.

Reply:
Ok. `libretime_watch` is being used a bunch of places so I'll have to change that to `libretime-watch` and run some tests to make sure I don't break anything. There's a second script that needs to run, `start_watching.py` which actually triggers a scan of the watched directories. I could change this to be `libretime-watch-trigger` instead of `libretime_watch_start`.

Reply:
Sounds good to me